docs/how-to-guides/blob-migration.md

```diff
@@ -0,0 +1,50 @@
+# Blob migration
+
+Currently DICOM files are stored with DICOM UIDs as blob names in blob storage, using the template `{account}/{container}/{studyUid}/{seriesUid}/{sopInstanceUid}_{watermark}.dcm`. 
+Since UIDs may include personal information about the context of their creation, such as patient information or identifiers. So we made the decision to change the way that we store DICOM files. In the next sections we have detailed steps for each scenario to migrate your existing blobs from old format to new format.
+
+## Blob migration configuration
+Below is the appsettings configuration for blob migration. Few properties needs to be updated to trigger migration.
+
+```json
+"DicomServer": {
+    "Services": {
+      "BlobMigration": {
+        "FormatType": "Old",
+        "StartCopy": false,
+        "StartDelete": false,
+        "CopyFileOperationId": "1d4689da-ca3b-4659-b0c7-7bf6c9ff25e1",
+        "DeleteFileOperationId": "ce38a27e-b194-4645-b47a-fe91c38c330f",
+        "CleanupDeletedFileOperationId": "d32a0469-9c27-4df3-a1e8-12f7f8fecbc8",
+        "CleanupFilterTimeStamp": "2022-08-01"
+      }
+}
+```
+
+## Migration steps
+
+1. If you are a new service and have not created any files, you can upgrade to the latest version of the service and skip the migration steps. That means you will see the BlobMigration.FormatType as "New" in appsettings configuration file. 
+
+2. If you have already uploaded DICOM files but does not care about previous data. You can use the [Delete API](../resources/conformance-statement.md#delete) to delete all the data which can free up space and then upgrade to the latest version of the service or you can upgrade to the latest version of the service and then use the [Delete API](../resources/conformance-statement.md#delete) to delete all the data.
```

Comment:
It might be simpler to have the new major version only aware of the new format, so any old format needs to be removed before the upgrade.

```suggestion
2. If you have already uploaded DICOM files but do not care about migrating the data, you can use the [Delete API](../resources/conformance-statement.md#delete) to delete all existing studies. You can then upgrade to the latest version of the service.
```